specs/MultiProfile.md

```diff
@@ -365,14 +355,21 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
 
 [uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
 interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
-  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
-  /// After the Delete() call completes, The render process of webviews on this profile will
-  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
-  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
-  /// for more details. The profile directory on disk will be actually deleted when the browser
-  /// process exits. Webview2 creation will fail with the HRESULT is ERROR_INVALID_STATE(0x8007139FL)
-  /// if you create it with the same name as a profile that is being deleted.
-  HRESULT Delete(); 
+  /// After the API is called, the profile will be marked for deletion. The
+  /// local profile’s directory will be tried to deleted at browser process
+  /// exit, if failure to delete, it will recursive try to delete at next
+  /// browser process start until successful.
+  /// The corresponding user's `ProfileDeleted` event handle function will
+  /// be triggered. After the function is triggered, continuing to use the
+  /// profile or its corresponding webviews is an undefined behavior.
+  /// After user's `ProfileDeleted` event handle function return or the user
```

Comment:
`ProfileDeleted` event is part of profile delete API, so we need to record it as well in this doc.

Reply:
Ok, has added it.